Material.Blazor/Components/DatePicker/InternalDatePickerYearButton.razor

```diff
@@ -1,8 +1,7 @@
 ﻿@namespace Material.Blazor.Internal
 @inherits ComponentFoundation
 
-<MBButton @attributes="@Attributes"
-          ButtonStyle="@ButtonStyle"
+<MBButton ButtonStyle="@ButtonStyle"
```

Comment:
We are the only ones using this component, hence we know there aren't any attributes to pass on.